website/src/components/canvas/Sidebar.jsx

```diff
@@ -43,19 +42,21 @@ export default class Sidebar extends React.Component {
                 open= {true}
                 docked={true}
                 openSecondary={false}
+                zDepth={2}
         >
-        <div>
+        <div style={styles.featureSpacing}>
           <h3>Rotate</h3>
-          <RotationSlider />
-          <Slider defaultValue={0.5} style={styles.slider}/>
+          <RotationSlider/>
           <h3>Resize</h3>
           <TextField
-            hintText="Enter new height"
-            floatingLabelText="Height: 64px"
+            hintText="Enter new height: 64px"
```

Comment:
Remove "Enter new height" entirely.
You can set the actual value of 64px through the defaultValue attribute